charts/zora/templates/plugins/trivy.yaml

```diff
@@ -58,6 +60,10 @@ spec:
         --all-namespaces \
         --scanners=vuln \
         -f=json \
+        --cache-dir=/tmp/trivy-cache \
```

Comment:
Are you adding a PVC/PV in this PR?  Are you keeping them separate?

Reply:
Done

Reply:
I'll do it in a separate PR